src/db.rs

```diff
@@ -198,6 +198,10 @@ impl Database {
                 debug!("output:\n{}", chunk);
             }
             if !column_names.is_empty() && !output.is_empty() {
+                if output.len() > 1 {
+                    let chunks: Vec<DataChunk> = output.drain(1..).collect();
+                    output[0].stack(&chunks);
+                }
                 output[0].set_header(column_names);
             }
             outputs.extend(output);
```

Comment:
We can replace `extend` by `push` now to make it more clear?

Reply:
well，i am not very familar with rust. 
I try this one first: `outputs.push(output[0])`,  and the complier warns me the DataChunk needs to implement the copy trait, i think copy the output[0] to outputs cost a lot. 
So here i replace it by `outputs.push(output.remove(0))` to move the output[0] to outputs. However, i think it is not elegant. 
Can you give me more suggestions? Any suggestions are welcome.